src/interface/tcpServer.c

```diff
@@ -147,7 +148,7 @@ void tcpServer_close(int socketDesc) {
 int tcpServer_getSocketConnect(int family) {
     int socketDesc;
     if ((socketDesc = socket(family, SOCK_STREAM, 0)) < 0) {
-        printf("Socket creation error.\n");
+        syslog(LOG_NOTICE, "Socket creation error.");
```

Comment:
Is the value of socketDescr potentially useful? If so, please include it in the log message.

Reply:
Log the socket in commit: ccc826f. Thanks!